resource.language.ro_ro/addon.xml

```diff
@@ -20,7 +20,9 @@
   </extension>
   <extension point="kodi.addon.metadata">
     <summary lang="en">Romanian language pack</summary>
+    <summary lang="ro">Pachet lingvistic pentru limba română</summary>
```

Comment:
No to translating them. There's no point. Please remove. Make sure only a single commit is left